nx/include/switch/kernel/ipc.h

```diff
@@ -45,9 +45,24 @@ typedef enum {
     IpcCommandType_Request = 4,
     IpcCommandType_Control = 5,
     IpcCommandType_RequestWithContext = 6,
-    IpcCommandType_ControlWithContext = 7
+    IpcCommandType_ControlWithContext = 7,
 } IpcCommandType;
 
+typedef enum {
+    DomainMessageType_Invalid = 0,
+    DomainMessageType_SendMessage = 1,
+    DomainMessageType_Close = 2,
+} DomainMessageType;
```

Comment:
Ipc prefix missing + elsewhere.

Reply:
I was matching the existing style -- @plutooo called the domain header type "DomainMessageHeader", does that also need to change?